src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -77,7 +78,8 @@ public TownyFlatFileSource(Towny plugin, TownyUniverse universe) {
 			dataFolderPath + File.separator + "nations.txt",
 			dataFolderPath + File.separator + "worlds.txt",
 			dataFolderPath + File.separator + "regen.txt",
-			dataFolderPath + File.separator + "snapshot_queue.txt"
+			dataFolderPath + File.separator + "snapshot_queue.txt",
+			dataFolderPath + File.separator + "metadata.yml"
```

Comment:
Is there still a metadata.yml or was that just in early versions?

Reply:
Just early versions